components/discovery/combiner/templates/index.html

```diff
@@ -23,7 +23,11 @@ <h2>Alliance information</h2>
         <div class="" style="min-height:200px;">
             <h2>Model performance</h2>
             <p>Track progress of the federated model.</p>
-              <a class="btn btn-primary" href="http://localhost:5111/box" role="button">Validations</a>
+
+            <div class="embed-responsive embed-responsive-21by9">
+                <iframe class="embed-responsive-item" src="http://localhost:5111/table" allowfullscreen></iframe>   
+            </div>
+            <a class="btn btn-primary" href="http://localhost:5111/box" role="button">Validations</a>
```

Comment:
This will only work intermittently in our test env. We need to refactor this out to a configurable host.

Reply:
Yep, there is an issue in Jira. We need to decide if we keep the Dashboard app separate as now, or if we integrate in the controller app or alliance_admin app.